dev-scripts/setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+root_dir=$(git rev-parse --show-toplevel)
+
+config_templates_dir="$root_dir/config-templates"
+config_dir="$root_dir/src/config"
+
+cp $config_templates_dir/.env.template $root_dir/.env
```

Comment:
Question: Should the `.env` file for Datadog config be in `config-templates` or the root dir? If it's supposed to be in the root dir should we move the `.env.template` file to the root dir?

Reply:
I created a `config-templates` directory to house all the configuration files that are mainly boilerplate files. The locations of where they should end up are somewhat enforced by the import linters and paths aliased in `tsconfig.json`. The setup script just helps automate the process of copying the boilerplates to the expected locations.

Think of the `config-templates` folder like the `templates` folder for our `Ansible` roles in `server-templates`